sass/_index.scss

```diff
@@ -1,335 +1,424 @@
+            @media (max-width: 767px) {
+                height: 15vh;
+            }
         }
 
-        .message {
-            color: #fff;
-            margin-bottom: 40px;
+        #hero_curve4 {
+            border-radius: 10% 100%;
+            background-color: #fff;
+            height: 25vh;
+            width: 200%;
+            position: absolute;
+            bottom: -7vh;
+            right: -50px;
+
+            @media (max-width: 767px) {
+                height: 15vh;
+            }
         }
     }
 
-    .button {
-        padding: 16px 32px;
-        border-radius: 32px;
-
+    #use-case-personal {
         background-color: #fff;
-        color: #000;
-    }
+        background: url("/assets/frontpage/personal-bg.svg");
+        background-size: 100% 100%;
+        @media (max-width: 767px) {
+            background-image: url("/assets/frontpage/personal-bg-mobile.svg");
+            background-size: 100% 241px;
+        }
+        background-repeat: no-repeat;
+        background-position-y: bottom -20px;
 
-    &:hover {
-        background-color: #0098d4;
-        color: #fff;
-    }
-}
+        position: relative;
+        overflow: hidden;
 
-@keyframes Gradient {
-    0% {
-        background-position: 0% 50%
-    }
+        display: flex;
+        flex-direction: row;
+        @media (max-width: 767px) {
+            flex-direction: column;
+        }
+        justify-content: space-around;
 
-    50% {
-        background-position: 100% 50%
-    }
+        padding-bottom: 175px;
 
-    100% {
-        background-position: 0% 50%
-    }
-}
+        h2 {
+            color: #000;
+            font-weight: 700;
+            margin: auto 0 auto 5rem;
+            @media (max-width: 767px) {
+                font-size: 34px;
+                margin: 0 50px 50px 50px;
+            }
+            max-width: 28rem;
+        }
 
-.dark-bg {
-    background-color: #f4f4f4;
-}
+        img {
+            margin: auto 0px;
 
-.imagine {
-    font-size: 17px;
-    font-weight: normal;
-    line-height: 1.5em;
-    text-align: left;
-    margin-left: auto;
-    margin-right: auto;
-    max-width: 400px;
-
-    ul {
-        list-style: none;
-    }
+            width: 469px;
+            height: 418px;
 
-    li {
-        text-indent: -.8em;
-        margin-left: 0em;
-        margin-bottom: .5em;
-    }
+            @media (max-width: 767px) {
+                margin: 0px auto;
+                width: 323px;
+                height: 288px;
+            }
+        }
 
-    h1 {
-        text-align: start;
-        padding: 0em;
-    }
-}
+        #personal_bottom {
+            position: absolute;
+            bottom: -340px;
+            width: 150%;
+            height: 400px;
+            background-color: black;
+            border-radius: 100% 100%;
 
-.thisismatrix {
-    h1 {
-        text-align: center;
-        font-weight: bold;
-        padding: 0em;
+            left: 50%;
+            transform: translateX(-50%);
```

Comment:
What does this do actually? I wonder if a transform is needed.

Reply:
This is used to centre the circle at the bottom of the div. I couldn't find another way to do it than with the `transform` that would work at all screens width. Is `transform` that expensive? It seems to be [widely supported](https://caniuse.com/?search=transform) now

Reply:
Centering is certainly not something I would expect a transform to be needed for, does `display: flex` + `justify-content` / `align-items` not work? Either way, not that important.

Reply:
It doesn't work because the item displayed is overflowing (with overflow hidden). In such a case `justify-content` and `align-items` seem to left-align the item, unfortunately. I've spent quite a bit of time scratching my head on this one :D

Reply:
Solved in d1e56d3